lib/pltr/v2/selectors/secondOrder.js

```diff
@@ -212,6 +222,45 @@ export const timelineViewIsntDefaultSelector = createSelector(timelineViewSelect
   return view !== 'default'
 })
 
+const selectedOrientationSelector = (state) => state.ui.orientation
+
+export const stickyHeaderCountSelector = createSelector(
+  pinnedPlotlinesSelector,
+  timelineViewIsStackedSelector,
+  hierarchyLevelCount,
+  selectedOrientationSelector,
+  (pinnedPlotlines, timelineViewIsStacked, hierarchyLevelCount, selectedOrientation) => {
+    if (selectedOrientation == 'vertical') {
+      return 1
+    }
+    if (timelineViewIsStacked) {
+      return hierarchyLevelCount
```

Comment:
This still doesn't seem right.  I think you should produce something like: hierarchyLevelCount + pinnedPLotlines + 1.
Is there a reason not to?